pyaedt/modeler/GeometryOperators.py

```diff
@@ -265,83 +271,84 @@ def v_cross(a, b):
     @staticmethod
     @aedt_exception_handler
     def _v_dot(a, b):
-        """ Evaluates the dot product between two geometry vectors a and b
+        """Evaluate the dot product between two geometry vectors.
 
         Parameters
         ----------
         a : list
-            first vector [x, y, z]
-
+            List of ``[x, y, z]`` coordinates for the first vector. 
         b : list
-            second vector [x, y, z]
```

Comment:
you can delete docstrings for private members (ones with _ as prefix in name). those methods will be ignored into documentation generation